engine/src/main/resources/assets/shaders/newParticle_frag.glsl

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015 Benjamin Glatzel <benjamin.glatzel@me.com>
```

Comment:
Fixed, will push with multiple renderers patch. I already replaced all of the @author / outdated headers. I missed the shaders since they're in the assets directory.